numba/cpython/builtins.py

```diff
@@ -575,6 +575,15 @@ def min_impl(indval1, indval2):
         return min_impl
 
 
+@overload(min)
+def boolval_min(val1, val2):
+    if isinstance(val1, types.scalars.Boolean) and \
+       isinstance(val2, types.scalars.Boolean):
```

Comment:
```suggestion
    if isinstance(val1, types.Boolean) and \
       isinstance(val2, types.Boolean):
```
suggest using the top level re-export for consistency.